### PR TITLE
[1] add final db to inventory-server

### PR DESCRIPTION
**Pull request by Paulooze**
Run `docker-compose up` in the `frinx-inventory-server` folder.
This will run a development postgres database.
Copy `.env.example` content to `.env` file in the same directory.
Run `yarn run dev`.
Now the API server is running using a development database.

### CODE REVIEW

File: packages/frinx-inventory-server/src/api.helpers.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  sw: string | null;
+  sw_version: string | null;
+  mac_address: string | null;
+  serial_number: string | null;
+  vendor: string | null;
+  uniconfig_zone: number | null;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  mount_parameters: Record<string, any> | null;
+  username: string | null;
+  password: string | null;
```
Comment (MarcoMruz) [inline packages/frinx-inventory-server/src/api.helpers.ts:21]: can we define these type properties as optional?

File: packages/frinx-inventory-server/docker-volumes/postgres/01-init.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    config json,
+    location VARCHAR(255),
+    model VARCHAR(255),
+    sw VARCHAR(50),
+    sw_version VARCHAR(50),
+    mac_address macaddr,
+    serial_number VARCHAR(50),
+    vendor VARCHAR(50),
+    uniconfig_zone INTEGER,
+    mount_parameters jsonb,
```
Comment (MarcoMruz) [inline packages/frinx-inventory-server/docker-volumes/postgres/01-init.sql:31]: is this type definition correct?

File: packages/frinx-inventory-server/src/api.helpers.ts (hunk 1 above)
Comment (Paulooze) [reply packages/frinx-inventory-server/src/api.helpers.ts:21]: Nope, there is a difference between `password?: string;` and `password: string | null`.

File: packages/frinx-inventory-server/docker-volumes/postgres/01-init.sql (hunk 2 above)
Comment (Paulooze) [reply packages/frinx-inventory-server/docker-volumes/postgres/01-init.sql:31]: I don't understand, it's an SQL schema, not a type definition.

Comment (MarcoMruz) [reply packages/frinx-inventory-server/docker-volumes/postgres/01-init.sql:31]: if jsonb is correct or it is typo

Comment (Paulooze) [reply packages/frinx-inventory-server/docker-volumes/postgres/01-init.sql:31]: No, it's not.

**Review by MarcoMruz — APPROVED** [review]

**Pull request merged**